package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "SAML-tracer",
+  "main": "index.js",
+  "repository": "https://github.com/simplesamlphp/SAML-tracer",
+  "license": "LGPL-2.1-or-later",
```

Comment:
Shouldn't this be [BSD-2 clause](https://github.com/simplesamlphp/SAML-tracer/blob/main/LICENSE)?

Reply:
Good catch! I'm not really sure, because all of our SimpleSAMLphp-repo's use `LGPL-2.1-or-later`.. No idea where that BSD-2 comes from

Reply:
Because it used to be in the UNINETT Github org and was unrelated to SSP.

Reply:
Is it safe to change it to LGPL?

Reply:
We would need 10 contributors to all agree, it's not impossible, but doesn't seem to solve anything specific I guess.